ADVOCATES.md

```diff
@@ -63,4 +63,5 @@ More information can be found in the [Chef Community Guidelines](https://github.
 * Vacant - Submit a PR to become a Chef Community Advocate
 
 #### Chef_Opscode Reddit ([https://www.reddit.com/r/chef_opscode/](https://www.reddit.com/r/chef_opscode/))
```

Comment:
Yep, I'd love to, but I don't know if it's possible, i chose `/r/chef_opscode` three years ago because `/r/chef` private.